akka-bbb-apps/src/main/scala/org/bigbluebutton/core/running/LiveMeeting.scala

```diff
@@ -5,23 +5,26 @@ import org.bigbluebutton.core.apps._
 import org.bigbluebutton.core.models._
 import org.bigbluebutton.core2.MeetingStatus2x
 
+import java.util
+
 class LiveMeeting(
-    val props:              DefaultProps,
-    val status:             MeetingStatus2x,
-    val screenshareModel:   ScreenshareModel,
-    val audioCaptions:      AudioCaptions,
-    val timerModel:         TimerModel,
-    val chatModel:          ChatModel,
-    val externalVideoModel: ExternalVideoModel,
-    val layouts:            Layouts,
-    val pads:               Pads,
-    val registeredUsers:    RegisteredUsers,
-    val polls:              Polls, // 2x
-    val wbModel:            WhiteboardModel,
-    val presModel:          PresentationModel,
-    val captionModel:       CaptionModel,
-    val webcams:            Webcams,
-    val voiceUsers:         VoiceUsers,
-    val users2x:            Users2x,
-    val guestsWaiting:      GuestsWaiting
+    val props:               DefaultProps,
+    val status:              MeetingStatus2x,
+    val screenshareModel:    ScreenshareModel,
+    val audioCaptions:       AudioCaptions,
+    val timerModel:          TimerModel,
+    val chatModel:           ChatModel,
+    val externalVideoModel:  ExternalVideoModel,
+    val layouts:             Layouts,
+    val pads:                Pads,
+    val registeredUsers:     RegisteredUsers,
+    val polls:               Polls, // 2x
+    val wbModel:             WhiteboardModel,
+    val presModel:           PresentationModel,
+    val captionModel:        CaptionModel,
+    val webcams:             Webcams,
+    val voiceUsers:          VoiceUsers,
+    val users2x:             Users2x,
+    val guestsWaiting:       GuestsWaiting,
+    val clientConfiguration: Map[String, Object],
```

Comment:
I suggest to name this prop as `clientSettings`.

Reply:
Agreed